docs_user/assemblies/assembly_migrating-ceph-rgw.adoc

```diff
@@ -0,0 +1,24 @@
+[id="migrating-ceph-rgw_{context}"]
+
+:context: migrating-ceph-rgw
+
+= Migrating {Ceph} RGW
+
+For hyperconverged infrastructure (HCI) or dedicated Storage nodes that are running {Ceph} version 6 or later, you must migrate the RGW daemons that are included in the {rhos_prev_long} Controller nodes into the existing external Red Hat Enterprise Linux (RHEL) nodes. The existing external RHEL nodes typically include the Compute nodes for an HCI environment or {Ceph} nodes. 
+
+To migrate Ceph Object Gateway (RGW), your environment must meet the following requirements:
+
+* {Ceph} is running version 6 or later and is managed by cephadm/orchestrator.
+* An undercloud is still available, and the nodes and networks are managed by {OpenStackPreviousInstaller}.
+//kgilliga: SMEs, FYI: In the downstream docs, the variable {OpenStackPreviousInstaller} = director.
```

Comment:
ack and thanks, it makes sense to refer to `director` instead of `TripleO` in the downstream doc.